front/components/vaults/EditVaultStaticDatasourcesViews.tsx

```diff
@@ -59,14 +59,15 @@ export function EditVaultStaticDataSourcesViews({
         }}
         className="absolute bottom-8 right-0"
       />
-      <VaultCreateFolderModal
+      <VaultUpsertFolderModal
```

Comment:
VaultCreateOrEditFolderModal ?

Reply:
Renamed! -> it will soon handle delete too, so called it VaultFolderModal (suggested by @q)